src/main/java/org/spongepowered/api/data/type/BodyPart.java

```diff
@@ -25,9 +25,10 @@
 package org.spongepowered.api.data.type;
 
 import org.spongepowered.api.CatalogType;
+import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 @CatalogedBy(BodyParts.class)
-public interface BodyPart extends CatalogType {
+public interface BodyPart extends CatalogType, Translatable {
```

Comment:
Okay, but how would that addition fit into the title of this PR, "Improve Texts.format(...) method"?